app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -159,7 +159,11 @@ class RootActivity : AppCompatActivity(), RootScreen {
                 onDatePicked ->
             val datePicker =
                 MaterialDatePicker.Builder.datePicker()
-                    .setSelection(MaterialDatePicker.todayInUtcMilliseconds())
+                    .setSelection(
+                        if(initialDate == null)
```

Comment:
Can you flip the `if (initialDate != null)` so the code is more readable

Reply:
made the change